client/src/App.js

```diff
@@ -69,7 +69,7 @@ function Guests() {
 
 function Calender() {
   return (
-   <Calender/>
+   <Calender2/>
```

Comment:
you don't need this function to render this `Calender2` component in the route, there is a lot like this in the file.

instead of wrapping the components in the function, just import and render them in the route.

also there is a lot of commented out code in this file that you don't really need i think, delete it